src/helper.js

```diff
@@ -6,9 +6,15 @@ export default class DistrictRepository {
       if (!acc[obj.Location]){
         acc[obj.Location]= [];
       }
-        acc[obj.Location].push(obj);
+        acc[obj.Location].push({ year: obj.TimeFrame,
```

Comment:
We may want to adjust what the objects being pushed in look like further. We could possible just push an object with the time frame i.e. the year for a key and the data as the value.